javascript/main.js

```diff
@@ -0,0 +1,52 @@
+const template = `
+  <h1 class="text-center mt-3">Featured Speakers</h1>
```

Comment:
- [ ] Great Job adding the Featured Speakers with JavaScript, However please store your Speakers information in an array of speaker objects that each holds their specific data, then loop through your array to create your template here:

### EXAMPLE
```
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```